templates/partials/candidates-filter.html

```diff
@@ -11,4 +11,5 @@
 {{ states.field('state') }}
 {% include 'partials/filters/districts.html' %}
 {% include 'partials/filters/candidate_status.html' %}
+{% include 'partials/filters/hideNull.html' %}
```

Comment:
I probably could have waited, but the new designs that arranged the tags, export buttons, and table info required this to move to a filter.